### PR TITLE
Fix for bug 406481 and 382982

### PR DESCRIPTION
**Pull request by tanvi-soni**
Fix for Bug 406481 and 382982. Changes the description string for "Visual Studio Test using Test Agent" and help drop down string for "Test Drop Location".

### CODE REVIEW

File: Tasks/RunDistributedTests/task.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
             "label": "Test Drop Location",
             "defaultValue": "",
             "required": true,
-            "helpMarkDown": "Location where the test binaries have been dropped in the agent machine(s) as part of the 'Copy files to Machines' task. System Environment Variables can also be used in location string. e.g., '%systemdrive%\\Tests', '%temp%\\DropLocation' etc.",
+            "helpMarkDown": "Location where the test binaries have been dropped in the agent machine(s) as part of the 'Copy files to Machine' or 'Azure File Copy' task. System Environment Variables can also be used in location string. e.g., '%systemdrive%\\Tests', '%temp%\\DropLocation' etc.",
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/task.json]: Windows Machine File Copy is the correct name of first task

File: Tasks/RunDistributedTests/task.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "id": "D353D6A2-E361-4A8F-8D8C-123BEBB71028",
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/task.json]: task.json.loc file is missing

**Comment by acesiddhu** [comment]
Looks fine

**Comment by bryanmacfarlane** [comment]
accept or close

**Comment by bryanmacfarlane** [comment]
accept or close

**Pull request closed without merging**